deploy/helm/flightctl/templates/flightctl-prometheus-clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
We should not install this on OCP, i.e.

{{ if .Values.prometheus.enabled }}
...
...